src/sites/dMCDA.jl

```diff
@@ -485,20 +545,10 @@ function guided_site_selection(
         SH, wsh = create_shade_matrix(A, max_area, w_shade_conn, w_waves, w_heat, w_predec_shade, w_predec_zones_shade, w_high_cover)
     end
 
-    if alg_ind == 1
-        mcda_func = order_ranking
-    elseif alg_ind == 2
-        mcda_func = topsis
-    elseif alg_ind == 3
-        mcda_func = vikor
-    else
-        error("Unknown MCDA algorithm selected. Valid options are 1 (Order Ranking), 2 (TOPSIS) and 3 (VIKOR).")
-    end
-
     if log_seed && isempty(SE)
         prefseedsites = zeros(Int64, n_site_int)
     elseif log_seed
-        prefseedsites, s_order_seed = rank_seed_sites!(SE, wse, rankings, n_site_int, mcda_func)
+        prefseedsites, s_order_seed = rank_sites!(SE, wse, rankings, n_site_int, mcda_func, 2)
```

Comment:
What do the "2" and "3" mean when used as an argument to `rank_sites!()`?

Reply:
These are the column used for storage in rankings for seeding and shading. In later changes (where mcda_vars , make_decision_matrix and make_seed_matrix/make_shade_matrix are replaced with more generic constructors) this is removed because rankings becomes generic. I thought I'd leave this change for the branch where I'm changing mcda_vars etc.

Reply:
This isn't a sustainable approach - we don't want you (and more importantly YOU won't want) to come back and add new numbers and `elseifs` every time a new criteria (or model with new criteria) gets added.

I suggest leaving this as is for now, we'll work out a more manageable approach later.

Reply:
Yeah, as I said, in other changes I've made this is removed, but it requires other changes which are probably best done as part of another issue.

Reply:
Okay, great thank you.